$OpenBSD$

Swapped out the default on-screen keyboard for something lightweight.

--- src/ipe/lua/prefs.lua.orig	Sat Jul  7 19:37:46 2012
+++ src/ipe/lua/prefs.lua	Sat Jul  7 19:37:52 2012
@@ -132,7 +132,7 @@ end
 
 -- How to start onscreen keyboard
 if config.platform == "unix" then
-  prefs.keyboard = "onboard &"
+  prefs.keyboard = "xvkbd &"
 else
   prefs.keyboard = nil
 end
